parley/src/layout/editor.rs

```diff
@@ -21,6 +21,21 @@ pub enum ActiveText<'a> {
     Selection(&'a str),
 }
 
+/// Opaque representation of a generation
+// Overflow handling: the generations are only compared,
+// so wrapping is usually fine, except that in the unlikely
+// event that exactly `usize::MAX` generations happen before the
+// application gets around to drawing `Generation(0)`.
+#[derive(PartialEq, Default, Clone, Copy)]
+pub struct Generation(usize);
+
+impl Generation {
+    /// Make it different from what it currently is
+    pub fn differentiate(&mut self) {
```

Comment:
I don't like this name since it makes me think of other (mathematical) things ... but I can see why you don't want to call it `increment` (although plenty of other generation counters do so).

Reply:
It's called `nudge` now. :+ )

Reply:
I think semantically it's a bit messy having this on this type, as end-users aren't really expected to use this.